lab1_task1d.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
 <html>
-
+<!--rev dev-->
 <head>
   <meta charset="utf-8">
   <title>Завдання 1.a</title>
```

Comment:
Watch the indentation. Choose 2 or 4 spaces, but then keep it stable. Below is a mess:

```html
  <ul>    <!-- 2 spaces -->
      <li>Земля    <!-- 4 spaces -->
          <ol>
              <li>Луна</li>
          </ol>
      </li>
      <li>
          Марс
          <ol start="2">
            <li>Фотос</li>
            <li>Деймос</li>
          </ol>
      </li>
         <li>    <!-- 7 spaces -->
          Уран
          <ol start="4">
              <li>Ариэль</li>
              <li>Умбриэль</li>
              <li>Титания</li>
              <li>Оберон</li>
              <li>Миранда</li>
                            

          </ol>
```

Reply:
No need to put extra blank lines before the closing tag, below is fine:
```html
<ol start="4">
    <li>Ариэль</li>
    <li>Умбриэль</li>
    <li>Титания</li>
    <li>Оберон</li>
    <li>Миранда</li>
</ol>
```